test/new-e2e/utils/credentials/store-aws.go

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package credentials
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/clients"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+)
+
+type awsStore struct{}
+
+func (s *awsStore) get(key string) (string, error) {
+	ssmClient, err := clients.GetAWSSSMClient()
+	if err != nil {
+		return "", err
+	}
+
+	output, err := ssmClient.GetParameter(context.Background(), &ssm.GetParameterInput{Name: &key, WithDecryption: true})
```

Comment:
Maybe this context should be passed into the function so that we can have a deadline for this function.